mailwatch.py

```diff
@@ -83,15 +83,23 @@ def poll(self, host, mailbox, user, passwd, room, ssl=True):
 			if mail.get('Message-ID') not in seen:
 				logging.debug("New message: {0}".format(mail.get('Message-ID')))
 				seen.append(mail.get('Message-ID'))
-				message = "New email arrived"
-				message += "\n\tFrom: {0}".format(email.header.decode_header(mail.get('from'))[0][0])
-				message += "\n\tTo: {0}".format(email.header.decode_header(mail.get('to'))[0][0])
-				message += "\n\tCc: {0}".format(email.header.decode_header(mail.get('cc'))[0][0])
-				message += "\n\tSubject: {0}".format(email.header.decode_header(mail.get('subject'))[0][0])
+
+				message = ''
+				for hdrname in ['From','To','Cc','Subject']:
+					value = mail.get(hdrname) or None
+					if value:
+						(hdrvalue, charset) = email.header.decode_header(value)[0]
+						if isinstance(hdrvalue, str):
+							message += "{}: {}\n".format(hdrname, hdrvalue)
```

Comment:
The original message put tabs (`\t`) in front of these lines to make it a little easier to visually distinguish multiple messages when there were multiple new messages. Would you mind adding the tabs back in?